pyrefinebio/config.py

```diff
@@ -28,6 +44,10 @@ def __new__(cls):
         return cls._instance
 
     def save(self, key, value):
+        """Save a value to the config
+
+        values that are saved are written to the config file at `~/.refinebio.yaml`
```

Comment:
Maybe we should say "... at \`os.getenv("CONFIG_FILE") or "~/.refinebio.yaml"`".